go/libraries/doltcore/table/untyped/xlsx/marshaling_test.go

```diff
@@ -63,10 +64,36 @@ func TestDecodeXLSXRows(t *testing.T) {
 
 func TestGetRows(t *testing.T) {
 	path := "test_files/employees.xlsx"
-	stateCols, _ := getXlsxRows(path, "states")
-	employeeCols, _ := getXlsxRows(path, "employees")
+	stateCols, _ := getXlsxRowsFromPath(path, "states")
+	employeeCols, _ := getXlsxRowsFromPath(path, "employees")
 
 	if stateCols != nil || employeeCols == nil {
 		t.Fatal("error")
 	}
 }
+
+func TestGetRowsFromBinary(t *testing.T) {
+	xlsxBinary := getBytesFromXlsx()
+	stateCols, _ := getXlsxRowsFromBinary(xlsxBinary, "states")
+	employeeCols, _ := getXlsxRowsFromBinary(xlsxBinary, "employees")
+
+	if stateCols != nil || employeeCols == nil {
+		t.Fatal("error")
+	}
+}
+
+func getBytesFromXlsx() []byte {
+	path := "test_files/employees.xlsx"
+
+	file, err := openFile(path)
```

Comment:
Does this work as just

```go
f, err := os.Open("test_files/employees.xlsx")
if err != nil {
    panic(err)
}
defer f.Close()
bs, err := ioutil.ReadAll(f)
if err != nil {
    panic(err)
}
return bs
```

Any reason to actually parse the xslx here?

Reply:
No, that works I fixed here: https://github.com/dolthub/dolt/pull/1053/commits/a19577ef29e2247f28c21f57084dd49fbefec4da

I think my googling led me astray 😁